vite.config.mts

```diff
@@ -18,6 +18,12 @@ const __dirname = dirname(fileURLToPath(import.meta.url))
 const cesiumSource = `${__dirname}/node_modules/cesium/Source`
 const cesiumWorkers = '../Build/Cesium/Workers'
 
+function manualChunks(id) {
+    if (id.includes('/src/utils/')) {
+        return 'utils'
+    }
+}
+
```

Comment:
Might be good to give a comment/summary of what purpose and effect this has on the final build, so that we may not need to go through many git commit log and/or Vite docu to figure out what's happening